src/t8_vtk.h

```diff
@@ -56,6 +56,11 @@
 #define T8_VTK_FORMAT_STRING "binary"
 #endif
 
+#if T8_WITH_VTK
+#define t8_vtk_locidx_array_type vtkTypeInt32Array
+#define t8_vtk_gloidx_array_type vtkTypeInt64Array
```

Comment:
Nice.

These should have a "_t" at the end as all t8code defined datatypes have.
```suggestion
#define t8_vtk_locidx_array_type_t vtkTypeInt32Array
#define t8_vtk_gloidx_array_type_t vtkTypeInt64Array
```